app/src/main/java/cm/aptoide/pt/theme/Theme.kt

```diff
@@ -0,0 +1,76 @@
+package cm.aptoide.pt.theme
+
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.darkColors
+import androidx.compose.material.lightColors
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.ReadOnlyComposable
+import androidx.compose.runtime.staticCompositionLocalOf
+import androidx.compose.ui.graphics.Color
+
+private val darkMaterialColorPalette = darkColors(
+    background = black,
+    onBackground = Color.White,
```

Comment:
does it make sense for background to be white on darktheme ?

Reply:
This can (and it should :) ) be changed in the future. At the moment this is not being used in the bottom navigation.